### PR TITLE
Users page only shows current user info. fix tests

### LINKED ISSUE #58: Users page shows everything

### PR DESCRIPTION
**Pull request by paymog**
fixes #58. Fix karma tests to pass

### CODE REVIEW

**Comment by kyleleung75** [comment]
I would say change the title of the button on the navbar to say "User" instead of "Users"

**Comment by paymog** [comment]
I'll change it to `Info`, I think that makes more sense. Good catch though.

File: webapp/views/partials/info.jade [hunk 1]
```diff
@@ -0,0 +1,15 @@
+div(class='container', ng-if='init()')
+    -var selected = 'Users';
```
Comment (kjrcda) [inline webapp/views/partials/info.jade]: this needs to be changed to info as well

Comment (paymog) [inline webapp/views/partials/info.jade]: Great catch.

**Pull request merged**